plugins/obs-outputs/rtmp-stream.c

```diff
@@ -1803,7 +1805,7 @@ struct obs_output_info rtmp_output_info = {
 #else
 	.encoded_video_codecs = "h264;av1",
 #endif
-	.encoded_audio_codecs = "aac",
+	.encoded_audio_codecs = "aac;flac;opus",
```

Comment:
This requires to update the `services.json` to not allow FLAC and Opus on every RTMP service.